src/components/MotionGenerator.tsx

```diff
@@ -1,19 +1,26 @@
 "use client";
 import { motion } from "@/types/motion";
 import motions from "@/data/motion.json";
-import { useEffect, useState } from "react";
+import { useContext, useEffect, useState } from "react";
 import { IconInfo } from "@/components/icons/info";
 import { useLang } from "@/lib/useLang";
 import { GenericButton } from "./GenericButton";
-import { IconRotateCCW } from "./icons/RotateCCW";
+import { LinkButton } from "./LinkButton";
+import { DebateContext } from "@/contexts/DebateContext";
+import { IconDice } from "./icons/Dice";
+import { IconClock } from "./icons/Clock";
 
 const MotionGenerator = () => {
   const [motion, setMotion] = useState<motion | null>(null);
   const infoslideString = useLang("infoslide");
+  const debateSetupPath = "/../oxford-debate/setup";
```

Comment:
I think it'd be better to use an absolute path (beginning with `/`).
Also no reason for this to be a variable.

Reply:
Sure, I'm gonna change to an absolute path. I did include it as a variable, as placing it directly in a prop seemed like creating a magic number.

Reply:
I mean, good instinct, but in this case the intent is perfectly represented by the path provided, don't you think? Hard to miss the origin of the string, unlike some ungodly Carmackian number like 0x5F3759DF.

(i love inverse square root tho)